lib/Widget/Countdown.php

```diff
@@ -71,7 +71,7 @@ public function installOrUpdate($moduleFactory)
             $module->regionSpecific = 1;
             $module->renderAs = 'html';
             $module->schemaVersion = $this->codeSchemaVersion;
-            $module->defaultDuration = 5;
+            $module->defaultDuration = 60;
```

Comment:
What triggered the change to the default duration?

Reply:
I just noticed ( while testing the video loop duration) that the default duration for an added countdown in the designer was 60 seconds, and the preview wasn't matching that value, should I revert this back to 5?